src/include/OSL/sfmath.h

```diff
@@ -94,7 +94,8 @@ namespace sfm
 
         // NOTE: negation happens outside of conditional, then is blended based on the condition
 #if OPENIMAGEIO_VERSION >= 20112
-        Dual2<float> neg_x = OIIO::fast_neg(x);
+        //Dual2<float> neg_x = OIIO::fast_neg(x);
+        Dual2<float> neg_x = fast_neg(x);
```

Comment:
It seems to me like this is fixing a separate bug (code in master is resetting derivs to 0 here, unless I'm missing something).

Maybe worth splitting into its own PR ?

I also think the `ifdef` is not needed since `fast_neg` is defined on duals inside OSL already.

Reply:
On second look, this code doesn't appear to be used at all. Maybe its safe to just nuke it?

Reply:
used in code you haven't seen in PRs yet

Reply:
Why did Cliff's PR have any changes at all to sfmath? Some sort of rebase confusion?

Reply:
Change has been reverted.  I'm having trouble building locally without it, however, and am not sure why.

Reply:
Let's take that offline, I'll give you a hand, it builds for me.